DataManagementSystem/Service/StorageElementHandler.py

```diff
@@ -50,6 +50,39 @@
 MAX_STORAGE_SIZE = 0
 USE_TOKENS = False
 
+def diskSpace(path, size = 'TB', total = False):
```

Comment:
According to DIRAC coding conventions, method name should start with a verb, e.g. getDiskSpace()